src/services/logging/index.ts

```diff
@@ -0,0 +1,32 @@
+import type { Log } from 'hyperview/src/types';
+
+export default class Logging {
```

Comment:
this doesn't seem to be in use?

Reply:
Not yet. I have some local changes which tested it out. I'll create separate stories to implement logging within Hyperview.

Reply:
I don't think we want Hyperview to implement a logger, more so use one if injected via props. This probably should remain outside of Hyperview core.

Reply:
This is just a helper within Hyperview to properly format and structure the data, auto-inject the current date (if none provided). The actual logger exists outside Hyperview.